scripts/promote_staging_to_prod.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+if [ ! -z $1 ]; then
+  resp=$(curl -s -H "Authorization: Bearer $1" "https://api.github.com/repos/chanzuckerberg/single-cell-data-portal/actions/runs?actor=czibuildbot" | jq -r '.workflow_runs[] | select(.display_title == "stage")')
+else
+  resp=$(curl -s  "https://api.github.com/repos/chanzuckerberg/single-cell-data-portal/actions/runs?actor=czibuildbot" | jq -r '.workflow_runs[] | select(.display_title == "stage")')
+fi
+
+checks=$(echo "$resp" | jq -r '.conclusion' | head -n 1)
```

Comment:
move the `head -n 1` operation back up to the end of the `curl` cmd, where it was before. We just want the `.conclusion` attribute grab on this line.

Reply:
this should work as-written, the `.conclusion` will grab the conclusion attribute on everything in the response list, then `head -n 1` will grab the first item in the list (most recent)

Reply:
Ah, sorry, my mistake. My suggestion was wrong. I agree it works as written but it is a bit indirect. My intended suggestion: